tools/cldr-code/src/main/java/org/unicode/cldr/draft/ScriptMetadata.java

```diff
@@ -139,6 +142,12 @@ public static void addNameToCode(String type, Map<String, String> hashMap) {
 
     public static final class SkipNewUnicodeException extends ICUException {}
 
+    /**
+     * Scripts that either have no known languages as yet (Cpmn) or are used for any language
+     * (Brai).
+     */
+    public static final Set<String> SCRIPTS_WITH_NO_LANGUAGES = Set.of("Brai", "Cpmn");
```

Comment:
"any language" would be good script metadata (and supplemental data).  Zyyy, Zxxx in the same boat.